.github/workflows/release.yml

```diff
@@ -9,18 +9,18 @@ on:
 
 jobs:
   build:
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+    uses: Netz39/Mitgliedsantraege/.github/workflows/ci.yml@main
```

Comment:
Ist das "main" hinten der Git Branch? Der heißt in diesem Repo "master".

Reply:
Guter Punkt. Wollen wir gleich das Repo auf main umbenennen?

Reply:
In dem Fall wäre eine allgemeine Information gut. Für alle, die das Repo ausgecheckt haben, weil das Auswirkung auf die Sandboxes hat.

Reply:
Als ich nach der Umbenennung auf die Hauptseite des Repos gegangen bin, kam eine Erinnerung von Github, dass der default Branch sich geändert hat.